src/screens/GroupInfo/elements/ItemLink.js

```diff
@@ -29,7 +29,7 @@ export default ItemLink;
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` seems to be a part of the color scheme update. Ensure that this new color is consistent with the overall design and accessible to all users, including those with visual impairments.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->